components/training/eventBooking.tsx

```diff
@@ -0,0 +1,224 @@
+import classNames from "classnames";
+import moment from "moment";
+import { FC } from "react";
+import { MdLocationOn } from "react-icons/md";
+import type { Template } from "tinacms";
+import { tinaField } from "tinacms/dist/react";
+import { EventBookingType, EventModel } from "./eventBookingType";
+
+const classes = {
+  mdColSpan4: "md:col-span-4",
+  mdColSpan6: "md:col-span-6",
+  mdColSpan12: "md:col-span-12",
+  lastMdColSpan4: "last:md:col-span-4",
+  lastMdColSpan8: "last:md:col-span-8",
+  lastMdColSpan12: "last:md:col-span-12",
+};
+
+export const EventBooking: FC<EventBookingType> = ({ data }) => {
+  return (
+    <>
+      {
+        <EventHeader
+          duration={data.duration}
+          price={data.price}
+          schema={data}
+        />
+      }
+      <div className="mb-2 grid grid-cols-12">
+        {data.eventList?.map((event, index) => (
+          <EventCard
+            key={index}
+            event={event}
+            count={data.eventList.length}
+            index={index}
+            schema={data}
+          />
+        ))}
+      </div>
+      <div className="bg-gray-400 py-1 text-center text-white">
+        {EventModel.HOSTED_BY}{" "}
+        <span className="font-bold">{EventModel.SSW}</span>
+      </div>
+    </>
+  );
+};
+
+const EventCard = ({ event, count, index, schema }) => {
+  return (
+    <div
+      className={classNames(
+        "col-span-12 gap-2 border-b-8 border-white  bg-gray-100 py-3 pl-5 text-lg last:border-b-0 last:border-r-0 md:border-b-0",
+        getColSpanClass(count, index),
+        addTopBorderForSecondRow(index),
+        addRightBorder(index)
+      )}
+    >
+      <span
+        className="font-bold capitalize"
+        data-tina-field={tinaField(
+          schema.eventList[index],
+          eventBookingBlock.eventList.city
+        )}
+      >
+        {event.city}
+      </span>
+      <div
+        className=" py-0.5 text-xs uppercase text-gray-500"
+        data-tina-field={tinaField(
+          schema.eventList[index],
+          eventBookingBlock.eventList.date
+        )}
+      >
+        {" "}
+        {event.date && moment(event.date).format("Do (ddd) MMMM YYYY")}
```

Comment:
Try use DayJS - We already have it installed 

https://chat.openai.com/share/ca2eab25-4e1e-4688-9be5-43a179a0ec4f

Reply:
done - replaced it